science/plumed/Portfile

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                plumed
+PortGroup           mpi 1.0
+PortGroup           linear_algebra 1.0
+PortGroup           debug 1.0
+
+version             2.3.1
+revision            0
+categories          science
+license             LGPL-3
+maintainers         gmail.com:giovanni.bussi openmaintainer
+description         PLUMED is a plugin for molecular dynamics
+long_description    PLUMED is a plugin for molecular dynamics that can be used \
+                    in combination with popular molecular dynamics codes to perform biased simulations. \
+                    Additionally, it can be used as a standalone tool to analyze trajectories.
+
+platforms           darwin
+
+homepage            http://www.plumed.org/
+
+master_sites        https://github.com/plumed/plumed2/releases/download/v${version}/
+distfiles           plumed-src-${version}.tgz
+
+checksums \
+   plumed-src-${version}.tgz \
+          sha256 0f0ec4998bd5fe9a897fdbecd26efb14859179bc7e9b43cb2c457c33cc64d5cf \
+          rmd160 569af66d66999a17e04c2912251a2135c7ee4f93 \
+   plumed-doc-${version}.tgz \
+          sha256 c9b0de509d5d974bad0b394bbcef7e041a6204e8d8da47d962f5b98ac7307912 \
+          rmd160 28bbe50a3a188419ce3ab1c09aff4f95bd0595c9 \
+
+# This is required so that patch made with "git format-patch" work correctly
+patch.pre_args -p1
```

Comment:
This is maintainer's choice. I usually delete `a/` and `b/` manually to avoid having to do this.